talend-component-maven-plugin/src/it/web/test/component_runtime_dev.json

```diff
@@ -0,0 +1,1289 @@
+                    "subject": "ResponseBody",
+                    "path": "content",
+                    "value": "200"
+                  },
+                  {
+                    "comparison": "Contains",
+                    "subject": "ResponseBody",
+                    "path": "content",
+                    "value": "<svg xmlns="
+                  }
+                ]
+              }
+            },
+            {
+              "entity": {
+                "type": "Request",
+                "method": {
+                  "link": "http://tools.ietf.org/html/rfc7231#section-4.3.1",
+                  "name": "GET"
+                },
+                "body": {
+                  "formBody": {
+                    "overrideContentType": true,
+                    "encoding": "application/x-www-form-urlencoded",
+                    "items": []
+                  },
+                  "bodyType": "Text"
+                },
+                "uri": {
+                  "query": {
+                    "delimiter": "&",
+                    "items": [
+                      {
+                        "enabled": true,
+                        "name": "language",
+                        "value": "en"
+                      },
+                      {
+                        "enabled": true,
+                        "name": "identifiers",
+                        "value": "${getEntityById(\"9387fb14-2d4d-40ac-bb9f-b9bc60476c75\").\"response\".\"body\".\"components\".\"0\".\"id\".\"id\"}"
+                      }
+                    ]
+                  },
+                  "scheme": {
+                    "name": "http",
+                    "version": "V11"
+                  },
+                  "host": "${\"server-ip\"}:${\"server-port\"}",
+                  "path": "/api/v1/configuration/detail/${getEntityById(\"9387fb14-2d4d-40ac-bb9f-b9bc60476c75\").\"response\".\"body\".\"components\".\"0\".\"id\".\"id\"}"
+                },
+                "description": "",
+                "id": "86e7a18a-fb5c-4251-970f-dbd01ecd3c95",
+                "name": "configurationtype details",
+                "headers": [],
+                "assertions": [
+                  {
+                    "comparison": "Equals",
+                    "subject": "ResponseStatus",
+                    "path": "code",
+                    "value": "200"
+                  },
+                  {
+                    "comparison": "Exists",
+                    "subject": "ResponseBody",
+                    "path": "content",
+                    "value": "200"
+                  }
+                ]
+              }
+            }
+          ]
+        }
+      ]
+    }
+  ],
+  "environments": [
+    {
+      "id": "b70b6814-bded-45e9-a3e0-dd8ab64fcc72",
+      "name": "DEMO2",
+      "variables": {
+        "11b2708d-a811-4d68-8570-30bdee5d0d53": {
+          "name": "name_input",
+          "value": "my name",
+          "enabled": true,
```

Comment:
needed?

Reply:
I have a look, it come from api tester, I don't know where it come from